--- node_modules/react-native-nitro-sound/nitrogen/generated/ios/swift/AudioSet.swift.orig
+++ node_modules/react-native-nitro-sound/nitrogen/generated/ios/swift/AudioSet.swift
@@ -218,7 +218,7 @@
   var AVLinearPCMBitDepthKeyIOS: AVLinearPCMBitDepthKeyIOSType? {
     @inline(__always)
     get {
-      return self.__AVLinearPCMBitDepthKeyIOS.has_value() ? self.__AVLinearPCMBitDepthKeyIOS.pointee : nil
+      return self.__AVLinearPCMBitDepthKeyIOS.pointee != nil ? self.__AVLinearPCMBitDepthKeyIOS.pointee : nil
     }
     @inline(__always)
     set {
